chapters/packaging.Rmd

```diff
@@ -315,37 +289,37 @@ Let's install our package in this virtual environment.\index{Python package!inst
 First we re-activate it:
 
 ```bash
-$ conda activate zipf
+$ conda activate pyzipf
 ```
 
 Next,
-we go into the upper `zipf` directory that contains our `setup.py` file
+we go into the upper `pyzipf` directory that contains our `setup.py` file
 and install our package:
 
 ```bash
-(zipf)$ cd zipf
-(zipf)$ pip install -e .
+(pyzipf)$ cd ~/pyzipf
+(pyzipf)$ pip install -e .
 ```
 
 ```text
-Processing /Users/amira/proj/py-rse/zipf/zipf
-Building wheels for collected packages: zipf
-  Building wheel for zipf (setup.py) ... done
-  Created wheel for zipf: filename=zipf-0.1.0-py3-none-any.whl
+Processing /Users/amira/pyzipf
+Building wheels for collected packages: pyzipf
+  Building wheel for pyzipf (setup.py) ... done
+  Created wheel for pyzipf: filename=pyzipf-0.1.0-py3-none-any.whl
     size=4574 sha256=b7d645f1d07775
   Stored in directory: /tmp/pip-ephem-wheel/wheels/a8/a6/0e/8b2a5cb
-Successfully built zipf
-Installing collected packages: zipf
-Successfully installed zipf-0.1.0
+Successfully built pyzipf
+Installing collected packages: pyzipf
+Successfully installed pyzipf-0.1.0
 ```
 
 The `-e` option indicates that we want to install the package in "editable" mode,
 which means that any changes we make in the package code are directly available to use
 without having to reinstall the package;
 the `.` means "install from the current directory".
 
-If we look in `~/opt/anaconda3/envs/zipf/lib/python3.8/site-packages/`,
-we can see the `zipf` package beside all the other locally-installed packages.
+If we look in `~/opt/anaconda3/envs/pyzipf/lib/python3.8/site-packages/`,
```

Comment:
Just a note that anaconda is in `opt` only for mac users. It would be in `~` directly on linux and in `C:\Users\Username` I believe on Windows. I don't know if we do/should mention this somewhere in the beginning what operating system we are on, I didn't see anything when searching for it. I think we currently have some differences between chapters, but not sure if they are noticeable enough to be confusing (e.g. I saw that the scripting package uses python on linux), probably not.

Reply:
In Chapter 2 we have a call-out box to explain why we are using `/Users/amira` as our home directory:

**Home Directory Variations**

Our home directory will be in different places on different operating systems. On Linux it may be `/home/amira`, and on Windows it may be `C:\Documents and Settings\amira` or `C:\Users\amira` (depending on the version of Windows). Our examples show what we would see on MacOS.

We could have one in this chapter that reads as follows?

**Anaconda location variations**

The default location for Anaconda differs between operating systems. Our examples show the default location on MacOS (`/Users/amira/opt/anaconda3`), but on Linux it is typically `/home/amira/anaconda3` and on Windows it might be `C:\Documents and Settings\amira\anaconda3` or `C:\Users\amira\anaconda3` (depending on the version of Windows). During the installation process users can also choose a custom location (Section 1.3).

Reply:
Yes that looks great!